third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompting-checker.js

```diff
@@ -278,6 +279,17 @@ UserPromptingChecker.prototype.promptUserForUnknownApp_ = function(
  */
 UserPromptingChecker.prototype.runPromptDialog_ = function(
     clientAppId, userPromptDialogData, promiseResolver) {
+  if (GSC.Packaging.MODE == GSC.Packaging.Mode.EXTENSION) {
+    chrome.windows.create({
```

Comment:
This code should live in GSC.PopupWindow.Server, not here. Here we just want to show the dialog without taking care of such low-level details.